src/plugins/data/common/index_patterns/index_patterns/index_pattern.ts

```diff
@@ -67,6 +73,7 @@ interface SavedObjectBody {
 
 type FormatFieldFn = (hit: Record<string, any>, fieldName: string) => any;
 
+const DATA_SOURCE_REFERNECE_NAME = 'dataSource_0';
```

Comment:
"_0" doesn't make sense to me as a constant value. Change to something else? Or just use the "type" as "reference name"？

Reply:
I am fine with `dataSource` then, unless a more suitable one comes up.